tests/test_automatic_payout_transactions.py

```diff
@@ -0,0 +1,89 @@
+from utils import stripe_obj_to_dict, client, midnight
+from tap_tester import runner, connections
+from base import BaseTapTest
+
+def get_payouts():
```

Comment:
Add functional comment to describe the reason for adding this function.

Reply:
Added comment.